certbot.wrapper

```diff
@@ -1,5 +1,32 @@
 #!/bin/bash
 set -e
+
+# This code is based on snapcraft's own patch to work around this problem at
+# https://github.com/snapcore/snapcraft/blob/a97fb5c7ea553a1bd20f4887a7c3393e75761890/patches/ctypes_init.diff.
+case "${SNAP_ARCH}" in
+    'arm64')
+        ARCH_TRIPLET='aarch64-linux-gnu';;
+    'armhf')
+        ARCH_TRIPLET='arm-linux-gnueabihf';;
+    'i386')
+        ARCH_TRIPLET='i386-linux-gnu';;
+    'ppc64el')
```

Comment:
We're probably not very likely to support other architectures, right? Maybe technically that means we should only have the ones we support here. But honestly I think it's cleaner and easier (for future changes) to keep them. That being said, it might be worth a note.

Reply:
We probably won't support other architectures, but we might. The situation is that with the exception of 32-bit x86, we trivially could support other architectures, but we currently don't because we haven't seen a demand for it.

Reply:
I added a comment.